main.py

```diff
@@ -249,12 +242,12 @@ def __init__(self, name, E, G, Iy, Iz, J, A):
 
     def __repr__(self):
         return ("Material: " + self.name + " [" + str(self.E) + ", " + str(self.G) +
```

Comment:
try f-strings to format this nicer.. like

`f"some text {a_variable} some more text {a variable}"`